app/views/debug/show.html.erb

```diff
@@ -0,0 +1,118 @@
+        event_dates = events.map(function(event){ return [event[0], event[1], event[2], new Date(event[3]), new Date(event[4])] });
+
+    event_timeline = new google.visualization.Timeline(document.getElementById('events_chart_div'));
+    event_table = createTimelineTable(eventColumns);
+    links_chart = new google.visualization.OrgChart(document.getElementById('links_chart_div')),
+    links_table = createTable([['base_path', 'string'], ['link', 'string'], ['content_id', 'string']]),
+
+    links_table.addRows(links);
+    states_table.addRows(states);
+    event_table.addRows(event_dates);
+
+    links_chart.draw(links_table, {allowHtml:true});
+    states_chart.draw(states_table, {allowHtml:true});
+
+    google.visualization.events.addListener(links_chart, 'select', selectLinksHandler);
+    google.visualization.events.addListener(event_timeline, 'select', selectHandler);
+
+    event_timeline.draw(event_table, {allowHtml:true, tooltip: {trigger: 'selection'}});
+  };
+
+  function selectLinksHandler(a,b,c,d) {
+    selection = links_chart.getSelection()[0].row;
+    data = links_table.getValue(selection, 0);
+  }
+
+  function selectHandler() {
+    selection = event_timeline.getSelection()[0].row;
+    data = event_table.getValue(selection, 2);
+    events = document.getElementsByClassName('events');
+
+    for (i = 0; i < events.length; i++) {
+      events[i].style.backgroundColor = "#fff";
+    }
+
+    el = document.getElementById(data);
+    el.style.backgroundColor = '#E2B4B4';
+    el.scrollIntoView();
+  }
+
+  function createTimelineTable(columns) {
+    var table = new google.visualization.DataTable();
+    columns.forEach(function(column) {
+      table.addColumn(column);
+    });
+    return table;
+  }
+
+  function createTable(columns) {
+    var table = new google.visualization.DataTable();
+    columns.forEach(function(column) {
+      table.addColumn(column[1], column[0]);
+    });
+    return table;
+  };
+</script>
+
+<div style="width: 90%; margin: auto;">
+  <h2 style='text-align:center'>links</h2>
+  <div id="links_chart_div"></div>
+  <h2 style='text-align:center'>states</h2>
+  <div id="states_chart_div"></div>
+  <h2 style='text-align:center'>events</h2>
+  <div id="events_chart_div" ></div>
+
+  <h3>Title</h3>
+  <div>
+    <%= @presenter.title  %>
+  </div>
+
+  <h3>Web</h3>
+  <%= link_to 'web', @presenter.web_url  %>
+
+  <h3>JSON</h3>
+  <%= link_to 'json', @presenter.api_url  %>
+
+  <h3>Current state</h3>
+  <%= @presenter.latest_state_with_locale %>
+
+  <h3>User facing verisons</h3>
+  <%= @presenter.user_facing_versions %>
+
+  <h3>Events</h3>
+  <% @presenter.events.each do |event| %>
+    <div class='events' id="<%= event.id  %>">
+      <p><b><%= event.action %></b>, <%= event.created_at.strftime("%Y-%m-%d %H:%I:%S") %><br />
+      <span class='payload' >
+        <% event.payload.each do |k,v| %>
```

Comment:
It would be possible to use the table helper from:

https://github.com/alphagov/search-admin/blob/master/app/helpers/table_helper.rb

That gives you nested tables for the data:

<img width="1226" alt="screen shot 2016-06-03 at 15 29 06" src="https://cloud.githubusercontent.com/assets/233676/15781867/eebf4f06-299f-11e6-8e6d-1ed64d2a8dfc.png">

You could also try:

```
      <pre class='payload'>
        <%= JSON.pretty_generate(event.payload) %>
      <pre>
```

To make it look like:

<img width="1204" alt="screen shot 2016-06-03 at 15 26 00" src="https://cloud.githubusercontent.com/assets/233676/15781836/c6099742-299f-11e6-9f27-b099124d0bb2.png">